### PR TITLE
DOCSP-31771 install landing page

### PR DESCRIPTION
**Pull request by davemungo**
## DESCRIPTION

[staging](https://docs-mongodbcom-staging.corp.mongodb.com/mongodb-shell/docsworker-xlarge/DOCSP-31771-install-landing-page/install/)

[JIRA](https://jira.mongodb.org/browse/DOCSP-31771)

[BUILD](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=64c407dc87b3b36046ef9d7b)

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/includes/fact-connect-prereqs.rst [hunk 1]
```diff
@@ -1,14 +1,14 @@
 To use the |mdb-shell|, you must have a MongoDB deployment to connect
 to. 
 
-- For a free cloud-hosted deployment, you can use
-  `MongoDB Atlas <https://www.mongodb.com/cloud/atlas?tck=docs_mongosh>`__.
+- For a free cloud-hosted deployment, you can use `MongoDB Atlas
```
Comment (davemungo) [inline source/includes/fact-connect-prereqs.rst:4]: Line break

File: source/includes/fact-connect-prereqs.rst [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
+  <https://www.mongodb.com/cloud/atlas?tck=docs_mongosh>`__.
 
 - To learn how to run a local MongoDB deployment, see
-  :manual:`Install MongoDB </installation/>`.
+  :ref:`Install MongoDB <tutorial-installation>`.
```
Comment (davemungo) [inline source/includes/fact-connect-prereqs.rst:8]: Update ref

**Review by davemungo — COMMENTED** [review]
Reviewer notes

**Review by mdb-ashley — APPROVED** [review]
LGTM! Thanks, Dave.

**Pull request merged**